sim/src/main/scala/endpoints/SimpleNICWidget.scala

```diff
@@ -24,42 +24,6 @@ import TokenQueueConsts._
 
 case object LoopbackNIC extends Field[Boolean]
 
-class SplitSeqQueue(implicit p: Parameters) extends Module {
```

Comment:
So why do we no longer need SplitSeqQueue? Did we resolve the issue with wide BRAM queues?

Reply:
A while back I changed them to use only one split; so they were basically just BRAMQueues under the hood. The firrtl bug has long since been resolved.